packages/ton-client/src/utils/pinch.ts

```diff
@@ -0,0 +1,51 @@
+import GestureDetector, { GESTURE_TYPES } from './gesture';
```

Comment:
also to be removed

Reply:
Weird, I think git's automatic merging strategy decided to keep it. While I haven't even touched these files...
All the gesture handling lives in a subdirectory `utils/gesture-detection` now, right?

Reply:
yes